.github/workflows/k8s-e2e.yaml

```diff
@@ -82,3 +82,21 @@ jobs:
     - run: kubectl apply -f  charts/example/centos-simple-kmod-0.0.1/centos-simple-kmod.yaml
     - run: sleep 360
     - run: kubectl logs `kubectl get pod -n openshift-special-resource-operator | grep special | awk '{ print $1 }'` -c manager -n openshift-special-resource-operator
+  remove:
+    name: remove-image
+    needs:
+      - build
```

Comment:
Technically it only needs `e2e` right?! I think we can remove the `build`.

Reply:
You are right. Updated.